doc/eredis.md

```diff
@@ -53,7 +53,7 @@ obfuscated() = fun(() -&gt; iodata())
 
 
 <pre><code>
-option() = {host, string() | {local, string()}} | {port, <a href="https://www.erlang.org/doc/man/inet.html#type-port_number">inet:port_number()</a>} | {database, integer()} | {username, iodata() | <a href="#type-obfuscated">obfuscated()</a> | undefined} | {password, iodata() | <a href="#type-obfuscated">obfuscated()</a> | undefined} | {reconnect_sleep, <a href="#type-reconnect_sleep">reconnect_sleep()</a>} | {connect_timeout, integer()} | {socket_options, list()} | {tls, [<a href="https://www.erlang.org/doc/man/ssl.html#type-tls_client_option">ssl:tls_client_option()</a>]} | {name, <a href="#type-registered_name">registered_name()</a>} | {sentinel, list()}
+option() = {host, string() | {local, string()}} | {port, <a href="inet.md#type-port_number">inet:port_number()</a>} | {database, integer()} | {username, iodata() | <a href="#type-obfuscated">obfuscated()</a> | undefined} | {password, iodata() | <a href="#type-obfuscated">obfuscated()</a> | undefined} | {reconnect_sleep, <a href="#type-reconnect_sleep">reconnect_sleep()</a>} | {connect_timeout, integer()} | {socket_options, list()} | {tls, [<a href="ssl.md#type-tls_client_option">ssl:tls_client_option()</a>]} | {name, <a href="#type-registered_name">registered_name()</a>} | {sentinel, list()}
```

Comment:
What happened here? These links will not work anymore.
How did you update this file?

Reply:
I just ran `make edoc` as it was instructed in the [contributing document](https://github.com/Nordix/eredis/blob/master/CONTRIBUTING.md). I think I was using and older rebar / Erlang version. I rerun `make edoc` with `rebar 3.23.0 on Erlang/OTP 25 Erts 13.0`, it looks okay now.

Reply:
Ah, ok.

I guess we need to change something to make it work correctly with newer versions. If you find out how, please let me know.